<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование профиля</string>
  </property>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_phone">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update_profile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Почта:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FIO">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select_photo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить фото</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
